src/test/java/org/sagebionetworks/bridge/exporter/record/BridgeExporterRecordProcessorTest.java

```diff
@@ -177,7 +177,7 @@ public void test() throws Exception {
         // validate that we cleaned up all our files
         assertTrue(mockFileHelper.isEmpty());
 
-        verify(mockDynamoHelper).bootstrapStudyIdsToQuery(eq(REQUEST));
+        verify(mockDynamoHelper).bootstrapStudyIdsToQuery(eq(REQUEST), any());
```

Comment:
Might be better to check for the actual endDateTime, since we know what it is in the test.